dbt_subprojects/daily_spellbook/models/_projects/gmx/event/arbitrum/gmx_v2_arbitrum_glv_shift_cancelled.sql

```diff
@@ -0,0 +1,167 @@
+{{
+  config(
+    schema = 'gmx_v2_arbitrum',
+    alias = 'glv_shift_cancelled',
+    materialized = 'incremental',
+    unique_key = ['tx_hash', 'index'],
+    incremental_strategy = 'merge'
+    )
+}}
+
+{%- set event_name = 'GlvShiftCancelled' -%}
+{%- set blockchain_name = 'arbitrum' -%}
+
+
+WITH evt_data_1 AS (
+    SELECT 
+        -- Main Variables
+        '{{ blockchain_name }}' AS blockchain,
+        evt_block_time AS block_time,
+        evt_block_number AS block_number, 
+        evt_tx_hash AS tx_hash,
+        evt_index AS index,
+        contract_address,
+        eventName AS event_name,
+        eventData AS data,
+        msgSender AS msg_sender
+    FROM {{ source('gmx_v2_arbitrum','EventEmitter_evt_EventLog1')}}
+    WHERE eventName = '{{ event_name }}'
+    {% if is_incremental() %}
+        AND {{ incremental_predicate('evt_block_time') }}
+    {% endif %}
+)
+
+, evt_data_2 AS (
+    SELECT 
+        -- Main Variables
+        '{{ blockchain_name }}' AS blockchain,
+        evt_block_time AS block_time,
+        evt_block_number AS block_number, 
+        evt_tx_hash AS tx_hash,
+        evt_index AS index,
+        contract_address,
+        eventName AS event_name,
+        eventData AS data,
+        msgSender AS msg_sender
+    FROM {{ source('gmx_v2_arbitrum','EventEmitter_evt_EventLog2')}}
+    WHERE eventName = '{{ event_name }}'
+    {% if is_incremental() %}
+        AND {{ incremental_predicate('evt_block_time') }}
+    {% endif %}
+)
+
+-- unite 2 tables
+, evt_data AS (
+    SELECT * 
+    FROM evt_data_1
+    UNION
```

Comment:
If these two CTEs are mutually exclusive, we may use `union all` here.
Otherwise, `union` is good here.

Reply:
Agree